packages/shared/src/components/feeds/FeedSettings/FeedSettingsCreate.tsx

```diff
@@ -0,0 +1,256 @@
+import { ActionType } from '../../../graphql/actions';
+
+const CreateTitle = ({ className }: { className?: string }): ReactElement => {
+  return (
+    <div className={classNames('flex items-center gap-4', className)}>
+      <Typography
+        className="tablet:typo-title3"
+        type={TypographyType.Body}
+        bold
+      >
+        New custom feed
+      </Typography>
+      <PlusUser />
+    </div>
+  );
+};
+
+const SubmitButton = (buttonProps: ButtonProps<'button'>): ReactElement => {
+  return (
+    <Button
+      {...buttonProps}
+      type="submit"
+      size={ButtonSize.Medium}
+      variant={ButtonVariant.Primary}
+    >
+      Create feed
+    </Button>
+  );
+};
+
+export const FeedSettingsCreate = (): ReactElement => {
+  const [newFeedId] = useState(() => Date.now().toString());
+  const { completeAction } = useActions();
+  const { user } = useAuthContext();
+  const router = useRouter();
+  const queryClient = useQueryClient();
+  const { displayToast } = useToastNotification();
+  const { logEvent } = useLogContext();
+  const [data, setData] = useState<CreateFeedProps>(() => ({
+    icon: '',
+  }));
+  const { createFeed } = useFeeds();
+  const { isPlus } = usePlusSubscription();
+
+  const { onFinished, delayedRedirect, isAnimating } = useProgressAnimation({
+    animationMs: 1000,
+  });
+
+  const { mutate: onSubmit, isPending: isSubmitPending } = useMutation({
+    mutationFn: createFeed,
+    onSuccess: (newFeed) => {
+      logEvent({
+        event_name: LogEvent.CreateCustomFeed,
+        target_id: newFeed.id,
+      });
+
+      queryClient.removeQueries({
+        queryKey: getFeedSettingsQueryKey(user, newFeedId),
+      });
+
+      onFinished();
+      delayedRedirect(`${webappUrl}feeds/${newFeed.id}`);
+    },
+    onError: (error) => {
+      const clientErrors = (error as ClientError)?.response?.errors || [];
+
+      if (
+        clientErrors.some(
+          (item) => item.message === labels.feed.error.feedLimit.api,
+        )
+      ) {
+        displayToast(labels.feed.error.feedLimit.client);
+
+        return;
+      }
+
+      if (
+        clientErrors.some(
+          (item) => item.message === labels.feed.error.feedNameInvalid.api,
+        )
+      ) {
+        displayToast(labels.feed.error.feedNameInvalid.api);
+
+        return;
+      }
+
+      displayToast(labels.error.generic);
+    },
+  });
+
+  useEffect(() => {
+    if (!user) {
+      return;
+    }
+
+    completeAction(ActionType.CustomFeed);
+  }, [completeAction, user]);
+
+  const logStartEventRef = useRef(false);
+
+  useEffect(() => {
+    if (logStartEventRef.current) {
+      return;
+    }
+
+    logStartEventRef.current = true;
+
+    logEvent({
+      event_name: LogEvent.StartCustomFeed,
+    });
+  }, [logEvent]);
+
+  const onRequestClose = () => {
+    router.replace(webappUrl);
+  };
+
+  useEffect(() => {
+    if (!isPlus) {
+      router.replace(webappUrl);
+    }
+  }, [isPlus, router]);
```

Comment:
In this version shouldn't it be something like you see the modal, but it's disabled and buttons are upgrade to plus?
(Like we do with new bookmarks now)

Reply:
Maybe, let me check, I though just show the entry point

Reply:
I think design removed it, I can't see it anymore, I'll log for checkup